src/style.css

```diff
@@ -1,3 +1,89 @@
-body {
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+* {
+  padding: 0;
   margin: 0;
+  border: 0;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+}
```

Comment:
- Don't use tag names for styling (except html and body)
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/46c247b8-2166-4794-a101-f0f8df79cf1c)